internal/pkg/stream/ecs.go

```diff
@@ -154,6 +161,35 @@ func (s *ECSDeploymentStreamer) Fetch() (next time.Time, done bool, err error) {
 			done = true
 		}
 	}
+
+	stoppedSvcTasks, err := s.client.StoppedServiceTasks(s.cluster, s.service)
+	if err != nil {
+		if request.IsErrorThrottle(err) {
+			s.ecsRetries += 1
+			return nextFetchDate(s.clock, s.rand, s.ecsRetries), false, nil
+		}
+		return next, false, fmt.Errorf("fetch stopped tasks: %w", err)
+	}
+	s.ecsRetries = 0
+
+	var stoppedTasks []ecs.Task
+	for _, st := range stoppedSvcTasks {
+		if stoppingAt := aws.TimeValue(st.StoppingAt); stoppingAt.Before(s.deploymentCreationTime) ||
+			(strings.Contains(aws.StringValue(st.StoppedReason), ecsScalingActivity)) {
+			continue
+		}
+		stoppedTasks = append(stoppedTasks, ecs.Task{
+			TaskArn:       st.TaskArn,
+			DesiredStatus: st.DesiredStatus,
+			LastStatus:    st.LastStatus,
+			StoppedReason: st.StoppedReason,
+			StoppingAt:    st.StoppingAt,
+		})
+	}
+	sort.SliceStable(stoppedTasks, func(i, j int) bool {
```

Comment:
Is there any reason that the sort needs to be a stable sort?

Reply:
Say if the `desiredcount` of the ECS service is 100, ECS scheduler tries to deploy all of these 100 tasks in parallel, And  there might be a good chance that one or more tasks will have the same `StoppingAt` time. So I just want to make sure that we have the same order of these tasks incase of equal `StoppingAt` time.